polkadot/xcm/xcm-builder/src/universal_exports.rs

```diff
@@ -438,16 +441,29 @@ impl<Bridge: HaulBlob, BridgedNetwork: Get<NetworkId>, Price: Get<MultiAssets>>
 		message: &mut Option<Xcm<()>>,
 	) -> Result<((Vec<u8>, XcmHash), MultiAssets), SendError> {
 		let bridged_network = BridgedNetwork::get();
-		ensure!(&network == &bridged_network, SendError::NotApplicable);
+		ensure!(&network == &bridged_network.0, SendError::NotApplicable);
 		// We don't/can't use the `channel` for this adapter.
 		let dest = destination.take().ok_or(SendError::MissingArgument)?;
-		let universal_dest = match dest.pushed_front_with(GlobalConsensus(bridged_network)) {
-			Ok(d) => d.into(),
-			Err((dest, _)) => {
-				*destination = Some(dest);
-				return Err(SendError::NotApplicable)
-			},
-		};
+
+		// Let's resolve the known/supported XCM version for the destination because we don't know
+		// if it supports the same/latest version.
+		let (universal_dest, version) =
+			match dest.pushed_front_with(GlobalConsensus(bridged_network.0)) {
+				Ok(d) => {
+					let version = Bridge::check_version_for(
+						&MultiLocation::from(AncestorThen(bridged_network.1, d)),
+						true,
```

Comment:
I've actually missed [here](https://github.com/paritytech/polkadot-sdk/pull/2481/files#r1415107234) that you pass `true` here too. Maybe we need to be extra cautious here and cut off everything below the relay chain/parachain level?